lib/src/components/NxTable/__tests__/NxTable.test.tsx

```diff
@@ -85,82 +91,369 @@ describe('NxTable', function() {
+      function ContextDependentChild() {
+        const { isFilterHeader } = useContext(RowContext);
+        return <span data-testid="context-dependent">{isFilterHeader.toString()}</span>;
+      }
+
+      const { rerender } = render(
+        <NxTable>
+          <NxTableBody>
+            <NxTableRow isFilterHeader={true}>
+              <NxTableCell>Foo</NxTableCell>
+              <NxTableCell>Bar</NxTableCell>
+              <NxTableCell />
+              <NxTableCell>Baz</NxTableCell>
+              <NxTableCell><ContextDependentChild /></NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      );
+
+      expect(screen.getByTestId('context-dependent')).toHaveTextContent('true');
+
+      rerender(
+        <NxTable>
+          <NxTableBody>
+            <NxTableRow isFilterHeader={false}>
+              <NxTableCell>Foo</NxTableCell>
+              <NxTableCell>Bar</NxTableCell>
+              <NxTableCell />
+              <NxTableCell>Baz</NxTableCell>
+              <NxTableCell><ContextDependentChild /></NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      );
+
+      expect(screen.getByTestId('context-dependent')).toHaveTextContent('false');
+
+      rerender(
+        <NxTable>
+          <NxTableBody>
+            <NxTableRow isFilterHeader={null}>
+              <NxTableCell>Foo</NxTableCell>
+              <NxTableCell>Bar</NxTableCell>
+              <NxTableCell />
+              <NxTableCell>Baz</NxTableCell>
+              <NxTableCell><ContextDependentChild /></NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      );
+
+      expect(screen.getByTestId('context-dependent')).toHaveTextContent('false');
+
+      rerender(
+        <NxTable>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>Foo</NxTableCell>
+              <NxTableCell>Bar</NxTableCell>
+              <NxTableCell />
+              <NxTableCell>Baz</NxTableCell>
+              <NxTableCell><ContextDependentChild /></NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      );
+
+      expect(screen.getByTestId('context-dependent')).toHaveTextContent('false');
+    });
+  });
+
+  describe('NxTableCell', function() {
+    const renderHeaderComponent = (extraProps?: NxTableCellProps) => render(
+      <NxTable>
+        <NxTableHead>
+          <NxTableRow>
+            <HeaderContext.Provider value={true}>
+              <NxTableCell { ...extraProps }/>
+            </HeaderContext.Provider>
+          </NxTableRow>
+        </NxTableHead>
+      </NxTable>
+    ).getByRole('columnheader') as HTMLElement;
+
+    describe('when the chevron prop is true', function() {
+      describe('when not isHeader', function() {
+        it('sets the button accessible name from the RowContext', function() {
```

Comment:
Again, the context is an impl detail.  Test that the accessible name gets set from the row text content and that it gets set from the `clickAccessibleLabel` prop

Reply:
> Again, the context is an impl detail. Test that the accessible name gets set from the row text content and that it gets set from the `clickAccessibleLabel` prop

Fixed it as part of: c54ac41c226c00680630c11145c503fe66946f46